roxie/ccd/ccdcontext.cpp

```diff
@@ -3838,12 +3838,6 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
         addWuExceptionEx(text, code, 2, MSGAUD_user, "user");
     }
 
-    virtual unsigned getWorkflowId() const override
```

Comment:
Use getWorkflowId from base class CRoxieContextBase.

Reply:
getWorkflowId in that class is:
```
    virtual unsigned getWorkflowId() { return 0; } // this is a virtual which is implemented in IGlobalContext
```

(meaning all fileservice scopes will be "w0:...." )


Looks like this change will avoid the roxie regression test failures, but doesn't seem right.

Unless there's a separate JIRA/PR to address this. If so, there should be some comments in the JIRA/PR to explain.

Reply:
getWorkflowId is defined as (line 1802)

    virtual unsigned getWorkflowId() const override { return graph->queryWorkflowId(); }
    
Where did you see getWorkflowId() defined to return 0?